src/mpi/coll/ialltoall/ialltoall_tsp_brucks_algos.h

```diff
@@ -363,11 +363,12 @@ MPIR_TSP_Ialltoall_sched_intra_brucks(const void *sendbuf, int sendcount, MPI_Da
     MPL_DBG_MSG_FMT(MPIR_DBG_COLL, VERBOSE,
                     (MPL_DBG_FDEST, "Step 3: data rearrangement scheduled\n"));
 
-  fn_exit:
     for (i = 0; i < nphases; i++) {
         MPL_free(tmp_sbuf[i]);
         MPL_free(tmp_rbuf[i]);
     }
+
+  fn_exit:
```

Comment:
What's the point of this change?

Reply:
I think this is @wesbland's patch.

Reply:
🤷 Drop it.

Reply:
I think in this case it can stay at the previous location so it frees the tmp_bufs too at exit. I will drop this commit.

Reply:
Dropped the patch and rebased the branch.